File: src/main/java/org/polyfrost/example/client/ExampleClient.java
package org.polyfrost.example.client;

import org.polyfrost.oneconfig.api.commands.v1.CommandManager;

public class ExampleClient {

    public static final ExampleClient INSTANCE = new ExampleClient();

    public void initialize() {
        ExampleConfig.INSTANCE.preload();
        CommandManager.register(new ExampleCommand());
    }

}

File: src/main/kotlin/org/polyfrost/example/client/ExampleClient.kt
package org.polyfrost.example.client

import net.fabricmc.api.ClientModInitializer
import org.polyfrost.oneconfig.api.commands.v1.CommandManager

object ExampleClient : ClientModInitializer {

    override fun onInitializeClient() {
        ExampleConfig.preload()
        CommandManager.register(ExampleCommand)
    }

}
